packages/cycle-scripts/template/config/javascript/flavor.js

```diff
@@ -0,0 +1,24 @@
+const dependencies = []
```

Comment:
Let's move this into the `/configs/flavor.js` perhaps now we could simply rename it into `dependencies.js` ?